lib/SodiumCrypto.js

```diff
@@ -590,4 +590,32 @@ export default class SodiumCrypto extends Crypto {
     // Returns a File object
     return decryptedBlob
   }
+
+  /**
+   * Helper function for tsv1 which wraps sodium's crypto_generichash function, uses BLAKE2b. https://libsodium.gitbook.io/doc/hashing/generic_hashing
+   *
+   * @returns {string} base64 encoded genericHash
+   */
+  async genericHash(stringToHash) {
+    await sodium.ready
+    let genericHash = sodium.crypto_generichash(
+      sodium.crypto_generichash_BYTES,
+      stringToHash
+    )
+    return this.b64encode(genericHash)
+  }
+
+  /**
+   * Helper function for tsv1 which wraps sodium's cyrpto_sign_detached fuction, https://libsodium.gitbook.io/doc/public-key_cryptography/public-key_signatures.
```

Comment:
The sdk will export `helpers` for the user.  So it might be worth avoiding use of the term to mean something like crypto helpers.